From f7c1b9d4e18dc650dd0b9da97a91f27f6a4bb5b7 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Sun, 17 Mar 2013 07:18:55 -0700
Subject: [PATCH] hostapd: change conf file for better sdk out of box startup

Change the default channel to 6 and change the ssid to a more
meaningful name.

Upstream-status: inappropriate [SDK specific patch]

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 hostapd/hostapd.conf |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hostapd/hostapd.conf b/hostapd/hostapd.conf
index f71a881..00843aa 100644
--- a/hostapd/hostapd.conf
+++ b/hostapd/hostapd.conf
@@ -83,7 +83,7 @@ ctrl_interface_group=0
 ##### IEEE 802.11 related configuration #######################################
 
 # SSID to be used in IEEE 802.11 management frames
-ssid=test
+ssid=TexasInstruments_0001
 
 # Country code (ISO/IEC 3166-1). Used to set regulatory domain.
 # Set as needed to indicate country in which device is operating.
@@ -107,7 +107,7 @@ hw_mode=g
 # channel will need to be configured separately with iwconfig.
 # When set to 0, automatic channel selection will be engaged. A channel
 # will be selected from the desired hw_mode.
-channel=1
+channel=6
 
 # Automatic channel selection (ACS) whitelist
 # (default: not set)
-- 
1.7.9.5

